### PR TITLE
Issue #7583: Added example for default config of if else tokens

### PR DESCRIPTION
**Pull request by Vyom-Yadav**
Issue #7583: Added example for default config of if else tokens

### CODE REVIEW

**Comment by Vyom-Yadav** [comment]
@Abhishek-kumar09 Sorry for closing the other PR as I set up a new upstream for my branch and in earlier PR couldn't set up due to an error. Please review this addition if it's fine then I will add it for all tokens.

**Comment by Vyom-Yadav** [comment]
Github, rebase

**Comment by Vyom-Yadav** [comment]
Github, generate web site

**Comment by Vyom-Yadav** [comment]
@Abhishek-kumar09 ping

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java [hunk 1]
```diff
@@ -72,8 +72,24 @@
  * <pre>
  * &lt;module name=&quot;NeedBraces&quot;&gt;
  *   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_IF, LITERAL_ELSE&quot;/&gt;
+ *   &lt;property name=&quot;allowSingleLineStatement&quot; value=&quot;false&quot;/&gt;
```
Comment (Abhishek-kumar09) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java:75]: This is not the default property.
Default config is one without any property being explicitly set.
This is the default config: 
```xml
<module name="NeedBraces"/>
```
You have to look at the table and generate the exmples foro the same.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Comment by Vyom-Yadav** [comment]
@Abhishek-kumar09 I am really sorry but I have to close this one also as my branches are conflicting. Currently, in my repo, there are two branches by the same name IDK how, I will have to delete these branches and create a new branch and a new PR, sorry for the inconvenience.

**Comment by Abhishek-kumar09** [comment]
@Vyom-Yadav  Don't do github rebase on website only, You have to rebase things on your local setup also. It may be the reason behind conflicts.

**Comment by Vyom-Yadav** [comment]
> @Vyom-Yadav Don't do github rebase on website only, You have to rebase things on your local setup also. It may be the reason behind conflicts.

Ok will do it from now

**Pull request closed without merging**